classifyDataFormat.go

```
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"os"
+)
+
+// ClassifyDataFormat ...
+func ClassifyDataFormat(r io.ReadSeeker) string {
+	d := json.NewDecoder(r)
+	token, _ := d.Token()
+	token, _ = d.Token()
+	defer r.Seek(0, 0)
```

You are swallowing the error here. Better to run it "manually" rather than in defer